.gitignore

```diff
@@ -61,6 +61,7 @@ php_errors.log
 /tests/PHPUnit/proxy/piwik.js
 /tests/PHPUnit/proxy/plugins
 /tests/PHPUnit/proxy/tests
+/tests/resources/piwik.test.js
```

Comment:
When running the JS tracker tests we generate a merged version of piwikjs and put the file to piwik.test.js